fuzzing/extra/nfc_ndef.cmake

```diff
@@ -0,0 +1,19 @@
+include_guard()
+
+add_definitions(-DHAVE_NFC -DHAVE_NDEF_SUPPORT)
```

Comment:
Let's not add_definitions to the project.
Either use target_compile_definitions to add it only to nfc_ndef or make an interface target 'common' that you link your targets agains.